package.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "docusaurus-website",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "clear": "docusaurus clear",
+    "serve": "docusaurus serve",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "tsc": "tsc",
+    "lint": "npm run eslint:check && npm run prettier:check",
+    "eslint:check": "eslint . --ext .json,.js,.jsx,.ts,.tsx",
+    "prettier:check": "prettier --check .",
+    "prettier:fix": "prettier --write .",
+    "qa": "npm run tsc && npm run lint && npm run build"
```

You can consider different name, e.g. `verify`:
```suggestion
    "verify": "npm run tsc && npm run lint && npm run build"
```

You can ignore this if you don't agree 👍

---

I like this quality assurance shortcut ;) I would like to leave it as it is.